modules/ROOT/pages/backup-and-disaster-recovery.adoc

```diff
@@ -30,6 +32,7 @@ Outputs logs to a file.
 Backup contents:
 
 [%header%autowidth.spread]
+.Backup Contents
 |===
 | Information | Included in PCE Backup 
 | Databases | Y
```

Comment:
Add a new row at the very end:

| Monitoring Add-On Metrics | N

Reply:
Added.